src/mavsdk/core/fs_utils.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
```

Comment:
We should probably merge this file with https://github.com/mavlink/MAVSDK/blob/main/src/system_tests/fs_helpers.h but it can happen after this PR.